Makefile

```diff
@@ -165,24 +155,25 @@ upload:
 
 #==== RELEASE : ====
 
-rel: ChangeLog dist tag homepage upload
+rel: ChangeLog rel-tarballs tag homepage upload
 	@echo "If all is perfect, eventually call   'make cleanup-rel'"
 	touch $@
 
 
 ## NB: The rpm (SuSE, RH, FC) and debian packages are built *and* signed
 ##     by the down stream maintainers:
 .PHONY: buildrpm
-buildrpm: dist
+buildrpm: rel-tarballs
 	rpmbuild -ta --sign $(ESSDIR).tgz
 
-builddeb:
-	dpkg-buildpackage -uc -us -rfakeroot -tc
+.PHONY: test-buildrpm
+test-buildrpm: rel-tarballs
```

Comment:
Why is this .PHONY? Shouldn't it produce a file?

Reply:
[we good now](https://github.com/emacs-ess/ESS/pull/732#issuecomment-439958230)

Reply:
I believe all tests should be PHONY targets.